src/routes/big_number_card.svelte

```diff
@@ -3,13 +3,22 @@
 
 	export let title: string;
 	export let description: string;
+
+	function updateBackgroundGradient(event: MouseEvent) {
+		const rectangle = (event.target as HTMLElement).getBoundingClientRect();
+		const leftPercentage: string = ((event.clientX - rectangle.left) / rectangle.width * 100).toString();
+		const topPercentage: string = ((event.clientY - rectangle.top) / rectangle.height * 100).toString();
+		(event.target as HTMLElement).style.background = `radial-gradient(circle at top ${topPercentage}% left ${leftPercentage}%, #a3a3a3, #404040)`;
+	}
+
+	function removeBackgroundGradient(event: MouseEvent) {
+	    (event.target as HTMLElement).style.background = '#404040';
```

Comment:
do we have other content in `style`? could we simply use tailwind for this? something like `(event.target as HTMLElement).style = 'bg-COLOR-HERE'`

Reply:
Unfortunately we can't assign anything to `style` as it is a read-only property 😞 

I did discover, however, that I can simply use `style.background = ''` as I'm already using `bg-neutral-700` on the list of classes for the div, so the class takes effect again, after I remove the styling.

I've pushed this commit – https://github.com/OnePlusOneInc/oneplusoneinc.github.io/pull/6/commits/adda07faa3431210feea49356180906bafe843aa – with the changes, let me know if that's better 🙇